docs/_interface/Private-Network.md

```diff
@@ -0,0 +1,291 @@
+and TCP traffic on port 30303.
+
+The bootstrap node needs to know about its own IP address in order to be able to relay it
+others. The IP is set using the `--nat` flag (insert your own IP instead of the example
+address below).
+
+```shell
+geth --datadir data --networkid 15 --nat extip:172.16.254.4
+```
+
+Now extract the 'node record' of the bootnode using the JS console.
+
+```shell
+geth attach data/geth.ipc --exec admin.nodeInfo.enr
+```
+
+This command should print a base64 string such as the following example. Other nodes will
+use the information contained in the bootstrap node record to connect to your peer-to-peer
+network.
+
+```text
+"enr:-Je4QEiMeOxy_h0aweL2DtZmxnUMy-XPQcZllrMt_2V1lzynOwSx7GnjCf1k8BAsZD5dvHOBLuldzLYxpoD5UcqISiwDg2V0aMfGhGlQhqmAgmlkgnY0gmlwhKwQ_gSJc2VjcDI1NmsxoQKX_WLWgDKONsGvxtp9OeSIv2fRoGwu5vMtxfNGdut4cIN0Y3CCdl-DdWRwgnZf"
+```
+
+Setting up peer-to-peer networking depends on your requirements. If you connect nodes
+across the Internet, please ensure that your bootnode and all other nodes have public IP
+addresses assigned and both TCP and UDP traffic can pass the firewall.
+
+If Internet connectivity is not required or all member nodes connect using well-known IPs,
+we strongly recommend setting up Geth to restrict peer-to-peer connectivity to an IP
+subnet. Doing so will further isolate your network and prevents cross-connecting with
+other blockchain networks in case your nodes are reachable from the Internet. Use the
+`--netrestrict` flag to configure a whitelist of IP networks:
+
+```shell
+geth <other-flags> --netrestrict 172.16.254.0/24
+```
+
+With the above setting, Geth will only allow connections from the 172.16.254.0/24 subnet,
+and will not attempt to connect to other nodes outside of the set IP range.
+
+### Running Member Nodes
+
+With the bootnode operational and externally reachable (you can try `telnet <ip> <port>`
+to ensure it's indeed reachable), you can start more Geth nodes and connect them via the
+bootstrap node using the `--bootnodes` flag.
+
+To create a member node running on the same machine as the bootstrap node, choose a
+separate data directory (example: `data-2`) and listening port (example: 30305):
+
+```shell
+geth --datadir data-2 --networkid 15 --port 30305 --bootnodes <bootstrap-node-record>
+```
+
+With the member node running, you can check whether it is connected to the bootstrap node
+or any other node in your network by attaching a console and running `admin.peers`. It may
+take up to a few seconds for the nodes to get connected.
+
+```shell
+geth attach data-2/geth.ipc --exec admin.peers
+```
+
+### Clique: Running A Signer
+
+To set up Geth for signing blocks in proof-of-authority mode, a signer account must be
+available. The account must be unlocked to mine blocks. The following command will prompt
+for the account password, then start signing blocks:
+
+```shell
+geth <other-flags> --unlock 0x7df9a875a174b3bc565e6424a0050ebc1b2d1d82 --mine
```

Comment:
What other flags are needed here? Is this a member node that needs a bootnode specified?

Reply:
Mining can be done on any node, be it 'member' node or otherwise. This is just supposed to say you might need other flags such as `--networkid`, `--bootnodes`, or even `--rpc`.

Reply:
I understand `--networkid` and `--bootnodes` are kinda recommended, while `--rpc` is optional. So I'd be explicit with the flags here, so a user can copy-paste the command.

Reply:
The issue with this is that people won't see the important flags anymore. What I've tried to do with these examples is only adding the flags that the text talks about.

Reply:
How about

```shell
OTHER_FLAGS='--networkid 42 --bootnodes xyz'
geth $OTHER_FLAGS --unlock ...
```